lib/Controller/MappingController.php

```diff
@@ -62,12 +62,11 @@ public function __construct($appName,
 	/**
 	 * test the given ldap config
 	 *
-	 * @param string $ldap_clear_mapping subject, 'group' or 'user'
-	 * @param string $ldap_serverconfig_chooser config id // FIXME remove in JS, is unneeded
+	 * @param string $subject, 'group' or 'user'
+	 *
 	 * @return DataResponse
 	 */
-	public function clear($ldap_clear_mapping, $ldap_serverconfig_chooser = null) {
-		$subject = $ldap_clear_mapping; // TODO if possible make JS send as 'subject' right away
+	public function clear($subject) {
```

Comment:
just to confirm the ldap_serverconfig_chooser / id isn't needed (?)

Reply:
no, it isn't. We truncate the whole user/group table https://github.com/owncloud/user_ldap/blob/master/lib/Mapping/AbstractMapping.php#L259-L264